frame/staking/src/lib.rs

```diff
@@ -1221,6 +1344,41 @@ impl<T: Trait> Module<T> {
 		Self::bonded(stash).and_then(Self::ledger).map(|l| l.active).unwrap_or_default()
 	}
 
+	/// Make sure that the account corresponding with the given account-id is a validator.
+	// TODO: needs a storage item to keep the most recent validators. ATM this is WRONG.
+	pub fn is_current_validator(who: &T::AccountId) -> bool {
+		Self::current_elected().contains(who)
```

Comment:
That should not be necessary after #4200, you would just pass the list of validators `AccountId`s to `submit_signed`

Reply:
This is already gone.